Airbrakes/logger.py

```diff
@@ -14,21 +17,22 @@ class Logger:
     real time.
     """
 
-    __slots__ = ("csv_headers", "log_path", "log_queue", "running", "log_process")
+    __slots__ = ("csv_headers", "log_path", "log_process", "log_queue", "running")
 
     def __init__(self, csv_headers: list[str]):
         self.csv_headers = csv_headers
 
-        self.log_path = os.path.join("logs", "log1.csv")
+        log_dir = Path("logs")
+        log_dir.mkdir(parents=True, exist_ok=True)
+
+        # Get all existing log files and find the highest suffix number
+        existing_logs = list(log_dir.glob("log_*.csv"))
```

Comment:
Note the new file name scheme I'm using here, e.g. `log_1.csv` instead of `log1.csv`.

Reply:
cleaner than what i had